test/src/e2e.long/mempoolMinfee.test.ts

```diff
@@ -0,0 +1,127 @@
+// This file is part of CodeChain.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import * as chai from "chai";
+import { expect } from "chai";
+import * as chaiAsPromised from "chai-as-promised";
+import "mocha";
+import {
+    validator0Address,
+    validator1Address,
+    validator2Address
+} from "../helper/constants";
+import CodeChain from "../helper/spawn";
+
+chai.use(chaiAsPromised);
+
+describe("MemPoolMinFees", async function() {
+    const chain = `${__dirname}/../scheme/tendermint-int.json`;
+    const config1 = `test/src/config/mem-pool-min-fee1.toml`;
+    const config2 = `test/src/config/mem-pool-min-fee2.toml`;
+    const node1_with_min_pay_fee_150 = new CodeChain({
+        chain,
+        argv: [
+            "--engine-signer",
+            validator0Address.toString(),
+            "--password-path",
+            "test/tendermint/password.json",
+            "--force-sealing",
+            "--config",
+            config1
+        ],
+        additionalKeysPath: "tendermint/keys"
+    });
+    const node2_with_min_pay_fee_200 = new CodeChain({
+        chain,
+        argv: [
+            "--engine-signer",
+            validator1Address.toString(),
+            "--password-path",
+            "test/tendermint/password.json",
+            "--force-sealing",
+            "--config",
+            config2
+        ],
+        additionalKeysPath: "tendermint/keys"
+    });
+    const node3 = new CodeChain({
+        chain,
+        argv: [
+            "--engine-signer",
+            validator2Address.toString(),
+            "--password-path",
+            "test/tendermint/password.json",
+            "--force-sealing"
+        ],
+        additionalKeysPath: "tendermint/keys"
+    });
+
+    beforeEach(async function() {
+        await node1_with_min_pay_fee_150.start();
+        await node2_with_min_pay_fee_200.start();
+        await node3.start();
+
+        await node1_with_min_pay_fee_150.connect(node2_with_min_pay_fee_200);
+        await node1_with_min_pay_fee_150.connect(node3);
+        await node2_with_min_pay_fee_200.connect(node3);
+
+        await node1_with_min_pay_fee_150.waitPeers(2);
+        await node2_with_min_pay_fee_200.waitPeers(2);
+        await node3.waitPeers(2);
+    });
+
+    it("two nodes with different minimum pay costs1", async function() {
```

Comment:
There is actually three nodes. The title is not descriptive. You should change the title to "should be a node that accepts a tx whose fee is greater than the node's min fee criteria, and the tx should be propagated through the network" or some similar and concise one.

Reply:
Thank you, I've changed the code but forgot to change the title.

Reply:
Changed the title to a descriptive one.